servers/store/src/main/proto/table.proto

```diff
@@ -43,13 +43,41 @@ message HiveDatabase {
   bytes database = 1;
 }
 
+message Dialect {
+  oneof object_type {
+    DremioDialect dremio_dialect = 1;
+    HiveDialect hive_dialect = 2;
+    SparkDialect spark_dialect = 3;
+    TrinoDialect trino_dialect = 4;
+  }
+}
+
 message SqlView {
   string sql_text = 1;
```

Comment:
im not sure I understand what this field is for.

Reply:
While expanding a view,  this would tell the compiler what contenxt to use to resolve the underlying tables. Should it use the current context of Nessie or the context the view was defined on which is stored in the view definition. @laurentgo suggested this in one of the reviews.